src/Pelagos/Bundle/AppBundle/Resources/public/js/sidebyside.js

```diff
@@ -0,0 +1,95 @@
+var $ = jQuery.noConflict();
+
+$(document).ready(function()
+{
+    "use strict";
+    
+    $("#get-versions-button").click(function (){
+        var udi = $("input[name=udi]").val();
+        jQuery.ajax({
+            url: Routing.generate("pelagos_app_ui_sidebyside_getversions", {udi: udi}),
+            type: "POST",
+            data: {udi: udi},
+            context: document.body
+        })
+        .success(function(data) {
+            var select = $("select.version-select");
+            select.find("option").remove();
+            
+            $.each(data, function(index, item) {
+                var option = new Option(item.sequence, item.sequence);
+                $(option).data("udi", item.udi);
+                $(option).data("modificationtimestamp", item.modificationtimestamp);
+                $(option).data("status", item.status);
+                $(option).data("version", item.version);
+                $(option).data("modifier", item.modifier);
+                select.append(option);
+            });
+            
+            $(".right-version").find("select option:selected")
+                .prop("selected", false)
+                .next()
+                .prop("selected", "selected");
+            select.change();
+        })
+        .error(function() {
+            var n = new noty({
+                text: "UDI:" + udi + " not found!",
+                type: "error",
+                theme: "relax",
+                timeout: 3000,
+                animation: {
+                    open: {opacity: "toggle"},  // fadeIn
+                    close: {opacity: "toggle"}, // fadeOut
+                }
+            });
+            $("input[name=udi]").val("");
+        });
+    });
+    
+    $(".left-version").find("select").change(function() {
+        var version = $(this).find("option:selected").data("version");
+        var udi = $(this).find("option:selected").data("udi");
+        
+        $("#left").html($(".spinner div").html());
+        $(".udi-title").text(udi);
+        
+        $(this).parents("div.left-version")
+            .find(".submission-status")
+            .text($(this).find("option:selected").data("status"));
+        $(this).parents("div.left-version")
+            .find(".submission-modificationtimestamp")
+            .text($(this).find("option:selected").data("modificationtimestamp"));
+        $(this).parents("div.left-version")
+            .find(".submission-modifier")
+            .text($(this).find("option:selected").data("modifier"));
+        
+        $("#left").load("/pelagos-symfony/dev/mvde/sidebyside/getForm/" + udi + "/" + version, function() {
```

Comment:
Your url is hard-coded, so it is throwing a no route found  "NotFoundHttp" Exception

Reply:
Fixed it